pkg/tfbridge/detailed_diff.go

```diff
@@ -311,6 +311,7 @@ func makeSingleTerraformInput(
 		ApplyTFDefaults:       false,
 		Assets:                AssetTable{},
 		UseTFSetTypes:         true,
+		DropUnknowns:          true,
```

Comment:
makeSingleTerraformInput should be renamed to make it clear it is about SET elements only. This is the only place it's used but the name make it sound it is generic, which makes DropUnknowns: scary here.